packages/aws-cdk/test/integ/cli/app/app.js

```diff
@@ -117,6 +118,23 @@ class DockerStackWithCustomFile extends cdk.Stack {
 
 }
 
+class FailedStack extends cdk.Stack {
+
+  constructor(parent, id, props) {
+    super(parent, id, props);
+
+    new s3.Bucket(this, 'MyBucket1', {
+      bucketName: 'one-will-fail'
```

Comment:
Add deletionPolicy and use a very long and creepy name 

Bucket names are world global

Reply:
will do

Reply:
Can we not make an invalid CloudFormation deployment some other way, rather than involving S3?

For example, create a role and add an empty IAM Policy with `force: true`.

Reply:
Or just make an empty CfnResource which is an `AWS::IAM::Policy` with nothing else, will fail as well.

Reply:
That sounds better. I'll try it.